stable/external-dns/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: external-dns
-version: 2.19.1
+version: 2.20.1
```

Comment:
Hi,
 As this is a small change, please update only the patch number, not the minor number.

Reply:
Committed the change to bump patch over minor.